Rubberduck.VBEditor.VB6/SafeComWrappers/VB/Window.cs

```diff
@@ -35,13 +35,21 @@ public IntPtr IUnknown
                 {
                     try
                     {
-                        _unknown = Marshal.GetIUnknownForObject(Target);
+                        try
+                        {
+                            _unknown = Marshal.GetIUnknownForObject(Target);
+                        }
+                        catch
+                        {
+                            // If GetIUnknownForObject throws us here, we're fine - it just means that the wrapper itself is invalid.
+                        }
                         Marshal.Release(_unknown);
```

Comment:
I don't think it's appropriate to call `Marshal.Release` after an exception. I'd presume that in such case, the `_unknown` will be `IntPtr.Zero` or something like that and that would be might bad to try and release a null pointer.

Reply:
:facepalm:  @bclothier - I just realized that these aren't used anymore (I abandoned that approach as unreliable). Removing them from the interface instead.